src/app/components/PortraitVideoCarousel/index.styles.tsx

```diff
@@ -0,0 +1,119 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+
+export const PROMO_ITEM_WIDTH = 160; // fixed width for one promo item
+
+const styles = {
+  section: () =>
+    css({
+      width: '100%',
+    }),
+
+  heading: () =>
+    css({
+      marginBottom: '1rem',
+    }),
+
+  scrollContainer: () =>
+    css({
+      position: 'relative',
+      display: 'flex',
+      alignItems: 'center',
+    }),
+
+  scrollWrapper: ({ spacings }: Theme) =>
+    css({
+      display: 'flex',
+      overflowX: 'auto',
+      scrollSnapType: 'x mandatory',
+      gap: `${spacings.DOUBLE}rem`,
+      paddingBottom: `${spacings.FULL}rem`,
+      scrollBehavior: 'smooth',
+      WebkitOverflowScrolling: 'touch',
+      '&::-webkit-scrollbar': {
+        display: 'none',
+      },
+      scrollbarWidth: 'none',
+      flex: 1,
+    }),
+
+  image: () =>
+    css({
+      width: '100%',
+      height: 'auto',
+      objectFit: 'cover',
+      borderRadius: '0.25rem',
+      aspectRatio: '9/16',
+      display: 'block',
+    }),
+
+  promoItem: () =>
+    css({
+      scrollSnapAlign: 'start',
+      flex: `0 0 ${pixelsToRem(PROMO_ITEM_WIDTH)}rem`,
+      textDecoration: 'none',
+      display: 'block',
+      position: 'relative',
+      borderRadius: '0.25rem',
+      overflow: 'hidden',
+    }),
+
+  gradientOverlay: () =>
+    css({
+      position: 'absolute',
+      bottom: 0,
+      left: 0,
+      right: 0,
+      padding: '3rem 0.5rem 0.5rem',
+      background:
+        'linear-gradient(to top, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0.85) 40%, rgba(0, 0, 0, 0.3) 75%, rgba(0, 0, 0, 0) 100%)',
+      zIndex: 1,
+    }),
+
+  promoHeading: ({ fontVariants, palette }: Theme) =>
+    css({
+      position: 'relative',
+      color: palette.WHITE,
+      fontFamily: fontVariants?.sansBold?.fontFamily,
+      fontSize: '0.875rem',
+      zIndex: 2,
+      textDecoration: 'none',
+      '&:hover': {
+        textDecoration: 'underline',
+      },
+    }),
+
+  buttonGroup: () =>
+    css({
+      position: 'absolute',
+      top: '50%',
```

Comment:
Need a more dynamic way of setting the position of the `buttonGroup` (wrapper for the navigations buttons) as this won't work for various breakpoints and devices. Open to suggestions here

Reply:
What issue are you seeing? The position seems to scale ok for me when going down to different breakpoints (50% from the top).

Reply:
Admittedly not seeing any issues (via Chrome) that couldn't be resolved with media queries for different group breakpoints but I think @louisearchibald and myself felt that hard coding the position of the buttons with this wrapper might result in unwanted positions when we come to implementing the finalised design. If it seems okay we can keep it as is!